### PR TITLE
Added a Blog

### PR DESCRIPTION
**Pull request by NaviRocker**
Added a Blog to the Blog Section

### CODE REVIEW

**Review by nikit19 — COMMENTED** [review]
Please provide a preview link

**Comment by NaviRocker** [comment]
@nikit19 This is from the `Patch-5` Branch.

File: _data/blogs.yml [hunk 1]
```diff
@@ -44,9 +44,14 @@
...
   
 - url: https://naveenrocker.wordpress.com/2017/12/01/first-time-in-google-code-in-with-fossasia/
-  title: First Time in Google Code In with Fossasia
+  title: First Time in Google Code-In with FOSSASIA
   img: 1.jpeg
   author_name: Naveen Rajan
+  
+- url: https://naveenrocker.wordpress.com/2018/01/09/interview-with-isuru-abeywardana-a-long-time-contributor/
+  title: Interview with an Open Source Contributor
+  img: 7.jpg
```
Comment (anshumanv) [inline _data/blogs.yml:53]: Please use a different image.

**Review by 0x48piraj — CHANGES_REQUESTED** [review]
Do the required changes

**Review by ParthS007 — CHANGES_REQUESTED** [review]
@naveentherocker Please do the suggested changes and update your PR 👍

**Pull request closed without merging**